src/app/appointments/[id]/page.tsx

```diff
@@ -0,0 +1,20 @@
+import Editar from "../../components/Forms/FormEditAppointment";
+
+async function Editpage({ params }: { params: { id: string } }) {
+  const { id } = await params;
+  const response = await fetch(`http://localhost:3001/api/appointments/${id}`);
+  const appointment = await response.json();
+
+  if (!appointment) {
+    // return <NotFound />;
+    console.log("no encontrado");
+  }
+
+  return (
+    <div>
+      <Editar appointment={appointment} />
+    </div>
```

Comment:
```suggestion
     <Editar appointment={appointment} />
```

Why is the component called `Editar`? 🤔 

```suggestion
      <EditForm appointment={appointment} />
```